Describe --bootstrap-version parameter for devextreme-cli (#2616)
DevExpress/devextreme-documentation: concepts/Common/DevExtreme CLI/50 ThemeBuilder/10 Command-Line Arguments.md

Specifies a path to the folder that contains the DevExtreme `icons` and `fonts` folders. Use this argument if the icons and fonts are not in the same folder as your custom theme.

- `--remove-external-resources`     
When present, removes links to external resources, such as fonts. The theme will use local fallbacks instead.

- `--bootstrap-version`     

> Specifies bootstrap version if `--input-file` is SCSS file for Bootstrap version 4 or 5. If this arguments is not defined Bootstrap 4 will be used.

Reviewer: ```suggestion
Specifies Bootstrap version 4 or 5 if `--input-file` is a `.scss` file (default: `4`).
```
